amidesfahani/filament-tinyeditor
# fix conflict TinyEditor with filament on overlap modal

### What happened?

https://private-user-images.githubusercontent.com/48276043/270893652-e30e7564-39ef-4ebd-9a5f-be62a0a8de0c.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIwNzA0NzUsIm5iZiI6MTcyMjA3MDE3NSwicGF0aCI6Ii80ODI3NjA0My8yNzA4OTM2NTItZTMwZTc1NjQtMzllZi00ZWJkLTlhNWYtYmU2MmEwYThkZTBjLm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI3VDA4NDkzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2ODRkZDllYzkxMjNhZGRiMWM1OTAzOTU1ZDMxMDg3YzYwNWM0OGM4YzhjZWRiOTE0NDM4NjhmMjFhMmRlOTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6jqhn6__H0QJW4o1uloRZKrrIl5-DTLGDThCy5l3lIk

### How to reproduce the bug

1. Create an action with modal form.
2. Add TinyEditor field inside it.
3. Open the modal.
4. Click Link 🔗 toolbox
5. New modal with some field will appear
6. Click the field, it wont focus on the clicked field

### Package Version
2.0.4

### PHP Version
8.2

### Laravel Version
11.18.1

### Which operating systems does with happen with?
macOS

**Notes**
No response
